stdlib/stdlib.mli

```diff
@@ -30,9 +30,16 @@ module Pervasives : sig
     This module is included in the toplevel [Stdlib] module.
 *)
 
+(** {1 Option type} *)
+
+type nonrec 'a option = 'a option = None | Some of 'a
```

Comment:
Same here, we already have Stdlib.Option.t.  Do we also want to add Stdlib.option?